pa4/checker-pa4.ml

```diff
@@ -153,9 +164,9 @@ let rec get_super_classes input_class_name ast_list visited = begin
 		let ((cloc,_),inherits,_) = List.find(fun ((_,cname),_,_)-> input_class_name = cname) ast_list in
 		match inherits with
 		| None -> 
-			(*return object*) 
-			let class_object = ((cloc, "Object"), None, obj_features) in
-			[class_object];
+                        let obj_name = "Object" in
+                        let class_object = ((cloc, obj_name), None, obj_features) in
```

Comment:
I don't think that having the `obj_name` variable adds anything to this code.  I think that we should go back to just having the string in this expression instead of in a variable.